node/packages/websocket/src/connectionHandler.ts

```
@@ -0,0 +1,109 @@
+  protected sendError(message: string, subscriptionId: string): void {
+    this.websocket.send(JSON.stringify({ subscriptionId, type: 'error', message } as ErrorResponse))
+  }
+
+  private onMessage(event: WebSocket.MessageEvent): void {
+    try {
+      const payload: RequestPayload = JSON.parse(event.data.toString())
+
+      switch (payload.method) {
+        // browsers do not support ping/pong frame, handle message instead
+        case 'ping': {
+          return this.websocket.send('pong')
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)